### PR TITLE
change button typography letter spacing CVR-674

### PR DESCRIPTION
**Pull request by hobbitronics**
[CVR-674](https://itse.youtrack.cloud/issue/CVR-674/Improve-Button-typography)

## Changed
- typography
- Button letter spacing

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -72,23 +72,19 @@ export default {
...
       template: ({ files }) => {
-        const script = (files.js || [])
-          .map(({ fileName }) => `<script src='/${fileName}'></script>`)
-          .join('\n')
+        const script = (files.js || []).map(({ fileName }) => `<script src='/${fileName}'></script>`).join('\n')
 
-        const css = (files.css || [])
-          .map(({ fileName }) => `<link rel='stylesheet' href='/${fileName}'>`)
-          .join('\n')
+        const css = (files.css || []).map(({ fileName }) => `<link rel='stylesheet' href='/${fileName}'>`).join('\n')
```
Comment (mtompset) [inline rollup.config.js:77]: Is minified the standard? The previous version was kind of readable by comparison.

File: src/components/mdc/_typography.scss [hunk 2]
```diff
@@ -4,11 +4,12 @@
 :root {
   font-family: Source Sans Pro, Roboto, sans-serif;
   line-height: 1.4rem;
-  --mdc-typography-font-family: Source Sans Pro, Roboto, sans-serif;
+  --mdc-typography-font-family: Source Sans Pro, sans-serif;
```
Comment (mtompset) [inline src/components/mdc/_typography.scss:7]: No love for Roboto?! :(

File: rollup.config.js [hunk 3]
```diff
@@ -72,23 +72,19 @@ export default {
...
         return getHtml(script, css)
-      }
+      },
```
Comment (mtompset) [inline rollup.config.js:79]: Is this like commas at the end of lists in PHP?

**Review by forevermatt — APPROVED** [review]

File: rollup.config.js (hunk 3 above)
Comment (hobbitronics) [reply rollup.config.js:79]: It was just prettier changing the formatting. Javascript can go either way.

File: rollup.config.js (hunk 1 above)
Comment (hobbitronics) [reply rollup.config.js:77]: prettiers work.

File: src/components/mdc/_typography.scss (hunk 2 above)
Comment (merchako) [reply src/components/mdc/_typography.scss:7]: Poor Roboto. The browser's favorite sans serif is probably going to provide a better consistent experience anyway. This way, it's either _SIL brand consistency_ or _OS consistency_ (and not 2010s Google brand consistency!)

**Pull request merged**